util/devel/deprecationScript/replaceDeprecatedLines.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import os
+
+num_replaced = 0
+
+def replaceSingleInstance(line, deprecated, replacement, include_files):
+  # Extract the file path and line number
+  # print(line)
+  file_path, line_number = line.split(':')[:2]
+  # Check if the line number is actually a number or a letter line nnn
+  # only cast to int if it is a number
+  if line_number.isdigit():
+    line_number = int(line_number)
+  else:
+    return
```

Comment:
else return is unneeded

Reply:
Unless I'm wrong, if I didn't return, it would throw an error for line with `nnnn` in place of a line number and halt instead of skipping over it.

Reply:
Oh I was reading too fast, I forgot there was a whole rest of the function. Sounds good!

Reply:
I'll improve it with better nesting though, it looks very ugly right now haha. I can do better
```
if not line_number.isdigit():
    return
line_number = int(line_number)